hbp_knowledge/receptors/walsh2008.bel

```diff
@@ -76,28 +76,41 @@ SET Evidence = "We first assayed expression of a6b2 or a3b2 receptors by
 binding the agonist, 125 I-labeled epibatidine (Fig. 1, A and B), to
 stably transfected a6FLAGb2HA cells and transiently trans-
 fected a3FLAGb2HA cells."
+SET Confidence = "High"
 complex(a(MESH:"alpha6beta2 nicotinic acetylcholine receptor"),a(CHEBI:epibatidine)) -> act(a(MESH:"alpha6beta2 nicotinic acetylcholine receptor"))
 complex(a(HBP:"alpha-3 beta-2 nAChR"),a(CHEBI:epibatidine)) -> act(a(HBP:"alpha-3 beta-2 nAChR"))
+UNSET Confidence
 
 SET Evidence = "At 37 °C, 1 ⁄ 3 of a 6-cm
 culture of HEK cells stably expressing a4b2 receptors typically
 bound 0.67 pmol of 125 I-labeled epibatidine, whereas the same
 size culture of a6b2-and a3b2-expressing cells bound 0.031 
 0.009 and 0.005  0.002 pmol, respectively (Fig, 1, A and B)."
+SET Confidence = "High"
 complex(a(HBP:"alpha-4 beta-2 nAChR"),a(CHEBI:epibatidine))
+UNSET Confidence
 
 SET Evidence = "After nicotine treat-
 ment for 24 h at 30µM nicotine, 125 I-labeled epibatidine bind-
 ing to a6b2 receptors increased significantly at 37 and 30°C."
+SET Confidence = "Medium"
 #binding was used to characterize up-regulation of recceptor: more binding bc more receptor
+a(CHEBI:nicotine) -> complex(a(CHEBI:epibatidine),a(MESH:"alpha6beta2 nicotinic acetylcholine receptor"))
 a(CHEBI:nicotine) -> a(MESH:"alpha6beta2 nicotinic acetylcholine receptor")
```

Comment:
Would agree with you, but also delete the second statement, because only more binding but no upregulation of receptor ist mentioned in the evidence...
```suggestion

```